lib/space.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'database_connection'
+
+class Space
+  def self.create(name:, description:, price:)
+
+    result = DatabaseConnection.query("INSERT INTO spaces (name, description, price) VALUES ($$#{name}$$,$$#{description}$$,'#{price}') RETURNING id, name, description, price")
```

Comment:
github is showing $$ hashtag name $$

presumably these are single quotation marks

Reply:
The $$string$$ syntax is used in the case that the string value contains a single quote for an apostrophe or something, so that confuses the SQL query. So you can replace 'string' with $$string$$ instead!